src/views/messages/message-rows/follow-user.jsx

```diff
@@ -29,7 +29,7 @@ var FollowUserMessage = React.createClass({
                     values={{
                         profileLink: <a
                             href={profileLink}
-                            className="social-messages-profile-link"
+                            class="social-messages-profile-link"
```

Comment:
@Kenny2github just an FYI – `className` is a [convention of ReactJS](https://facebook.github.io/react/docs/dom-elements.html#classname), so switching to use `class` is not ideal if there's a need for this class

Reply:
@mewtaylor I was intending to switch it back to `className` but my brain farted

Reply:
@Kenny2github lol I know the feeling :)